src/expr/datatype.cpp

```diff
@@ -834,6 +839,12 @@ void DatatypeConstructor::setSygus(Expr op,
   d_sygus_pc = spc;
 }
 
+const std::vector<DatatypeConstructorArg>* DatatypeConstructor::getSelectors()
```

Comment:
I would call this getArgs() and not getSelectors().

Reply:
Done